src/components/Common/TanBackground.js

```diff
@@ -0,0 +1,5 @@
+import styled from 'styled-components'
+
+export default styled.section`
+  background: ${props => props.theme.colors.tanBg};
```

Comment:
I would use `${({ theme }) => theme.colors.tanBg` for consistency with the rest of the code. It's not a blocker to merge. I like consistency :)

Reply:
updated and the bluebackground component, nice one!